ui/app/api/instance/info/route.ts

```diff
@@ -0,0 +1,10 @@
+import { GetFlowHttpAddressFromEnv } from '@/rpc/http';
+
+export const dynamic = 'force-dynamic';
+
+export async function GET() {
+  const flowServiceAddr = GetFlowHttpAddressFromEnv();
+  return fetch(`${flowServiceAddr}/v1/instance/info`, {
```

Comment:
This file isn't needed, send request to `/api/v1/instance/info`

Reply:
sidebar component is client side

Reply:
left suggestion on SidebarComponent.tsx, see `ui/app/api/v1` for how requests get transparently proxied

Reply:
my bad, I see we have a proxy